sources/packages/backend/libs/integrations/src/services/disbursement-schedule/disbursement-schedule.models.ts

```diff
@@ -276,6 +276,10 @@ export enum ECertFailedValidation {
    * Student MSFAA associated with the disbursement is not signed.
    */
   MSFAANotSigned = "MSFAANotSigned",
+  /**
+   * Student MSFAA associated with the disbursement is not signed or is cancelled.
+   */
+  MSFAANotValid = "MSFAANotValid",
```

Comment:
If I am following correctly `MSFAANotValid` is an combination between `MSFAANotSigned` and `MSFAACanceled`.
This layer should not be concerned about this or the API. If it is to adjust a message for the UI it should be handled at the UI layer only.